src/pages/blog/index.tsx

```diff
@@ -127,6 +208,7 @@ const Blog: React.FC<BlogProps> = ({ blogs, tags }) => {
       <BlogsHeader
         tags={tags}
         currentTag={tagQuery}
+        onLeaveInput={updateSearchParam}
```

Comment:
If I understand correctly, the query is only updated when the users leaves the input?

It doesn't matter that much, but I think it would be fitter to add that logic to the `onSearchChange` callback below so it updates in real time. It could also be debounced but I doubt a user can type fast enough for it to be a problem?

Reply:
I understand your point and thought about doing so, but I think its a preventable workload due to the fact that you need to leave focus in order to do anything else which will depend on these parameters